pkg/errors/errors.go

```diff
@@ -131,6 +131,21 @@ func IsTransient(err error) bool {
 	}
 }
 
+// IsRegistryTransient returns true if err represents a transient registry error
+func IsRegistryTransient(err error) bool {
```

Comment:
can we instead put this in the `registry`  package, so the call is something like `registry.IsTransientError(err error)` or something like that? that way we keep the registry-related business logic in a single package.